.travis.yml

```diff
@@ -26,25 +24,10 @@ services:
   - postgresql
 
 addons:
+  postgresql: "9.5"
```

Comment:
This is great! 😄

Reply:
I assume `mysql` is built in already?

Reply:
`postgresql` too but somehow it will be `2.3` :confused: 
So I guess we need to specify a newer version.

Reply:
`2.3` is the postgis version. I think explicit definition is better so that we know exactly what database versions and extension versions we are using. 😃

Reply:
No idea  how to specify version of `mysql` though.

Reply:
I tried a few things and they all failed. I don't really like to pin versions unless we don't have a choice. Without pinning we can catch errors early when something is updated.